tools/src/main/java/org/apache/kafka/tools/MetadataQuorumCommand.java

```diff
@@ -92,7 +92,7 @@ static void execute(String... args) throws Exception {
                 if (!commandConfig.exists())
                     throw new TerseException("Properties file " + commandConfig.getPath() + " does not exists!");
 
-                Utils.loadProps(commandConfig.getPath());
+                props = Utils.loadProps(commandConfig.getPath());
```

Comment:
How about making `props` final with `final Properties props;`? Setting here and added an `else` block that sets it to `new Properties()`.